packages/theme/components/HeaderNavigation/DesktopMenu.vue

```diff
@@ -33,7 +33,7 @@ export default {
     }
   },
   setup(props) {
-    const isMenuAvailable = computed(() => !props.menu.isDisabled);
+    const isMenuAvailable = computed(() => props.menu && !props.menu.isDisabled);
```

Comment:
Is this additional `props.menu &&` coming from changes in vuestorefront-core, or is that an additional check that you've added?

Reply:
I've added it - `props.menu` is falsy when menu is enabled in the app's config but there's no `Main Menu` added in Spree.

Reply:
We should probably extract it to a separate PR, right?

Reply:
Yeah, that would be preferred

Reply:
Done: https://github.com/vuestorefront-community/spree/pull/289